instructors/instructor-notes.md

```diff
@@ -89,40 +89,40 @@ polygon edge not found
 > {: .error}
 > ```
```

Comment:
Looks like something went funky with this error message block when the lesson was converted to the Workbench. I will open a PR to fix.